contributors/devel/release/releasenotes.md

```diff
@@ -0,0 +1,94 @@
+# Creating PRs for release notes
+
+This document describes how to create PRs in Kubernetes repos so that the changes can be tracked with the
+Kubernetes release and published in release notes.
```

Comment:
FWIW, I think it would make sense to group things better in alpha/beta/rcs as well